m4_sessions/1808-inning/surprise_of_happiness.md

```diff
@@ -24,9 +36,17 @@
      All sessions NEED to have a closing activity, reflection, discussion
     
 ## Introductons 
+
+     name, mod, and "What do you think makes you happy?"
+
 ## Intro to new material
+
 ## Small Group Activity
-## Closing Discussion
 
+     Have people line up by if they are a cat person, dog person, or both and then dance partner them so cat people are      working with dog people. 
+
+## Closing Discussion
 
+     Talk about what actually makes you happy and what are you going to do moving forward during your Turing career that will make you happy.
```

Comment:
Ahhh!  I see the closing here!  Great.  Are they going to share whole group or in pairs?  Maybe you can also share what y'all learned or things you are doing differently because of research you did for the session